lib/block-supports/elements.php

```diff
@@ -13,6 +13,11 @@
  * @return string                Filtered block content.
  */
 function gutenberg_render_elements_support( $block_content, $block ) {
+
+	if ( ! $block_content ) {
+		return;
```

Comment:
```suggestion
		return $block_content;
```

I think we should return content instead of void/nothing.

Reply:
I just want to understand why -If the condition is triggered, then there is no `$block_content` to return?

Reply:
The next callback might be expecting a string value even if it's empty. This changes the return type.

Here's similar example from layout support:
https://github.com/WordPress/gutenberg/blob/53374ba92c379480659a7df5ac456756ad3a798c/lib/block-supports/layout.php#L111-L117